### ISSUE TITLE
Add Convolution Neural Network

### ISSUE THREAD

**Issue by prathimacode-hub** [Algo-Phantoms/Algo-ScriptML#202]
I would like to add a simple convolution neural network to classify CIFAR images using Keras.

Can you assign this issue to me under 'GSSOC 21' label? @geekquad

**Comment by geekquad** [comment]
Hey @prathimacode-hub, please go through the project description.
We are looking for implementations of machine learning models and algorithms using nothing but NumPy with a focus on accessibility.

**Comment by geekquad** [comment]
However, you can look out for more issues you can work on. We have quite a few of them left @prathimacode-hub.

**Comment by prathimacode-hub** [comment]
I haven't notice that's it's purely on numpy. Yeah, I did checked them up. But I just came to know that today is the last day to submit the PR. @geekquad

**Comment by geekquad** [comment]
> I haven't notice that's it's purely on numpy. Yeah, I did checked them up. But I just came to know that today is the last day to submit the PR. @geekquad

It tomorrow till 11:59pm @prathimacode-hub.

**Comment by prathimacode-hub** [comment]
Ok, thanks for the information. I will choose an issue to work on. @geekquad
